jest-common/src/main/java/io/searchbox/core/SearchResult.java

```diff
@@ -103,6 +103,10 @@ public SearchResult(Gson gson) {
 
             if (source != null) {
                 JsonElement id = hitObject.get("_id");
+                String index = hitObject.get("_index").getAsString();
+                String type = hitObject.get("_type").getAsString();
+                Double score = hitObject.get("_score").getAsDouble();
```

Comment:
`score` field should be checked if it exists first (`hitObject.has()`) and also should be checked if it has null value (`hitObject.get().isJsonNull()`) and only then it is safe to do `getAsDouble()`.